From <https://sources.debian.org/data/main/p/ploticus/2.42-8/debian/patches/fix-ftbfs-gcc14.patch>.

Fixes a build error with GCC 14:

  svg.c: In function ‘PLGS_trailer’:
  svg.c:743:46: error: passing argument 1 of ‘gzprintf’ from incompatible pointer type [-Wincompatible-pointer-types]
    743 |                 if( svg_compress ) gzprintf( outfp, "%s", buf );
	|                                              ^~~~~
	|                                              |
	|                                              FILE *

--- a/src/svg.c
+++ b/src/svg.c
@@ -740,7 +740,7 @@
 	buf = svg_style; /* reuse */
 	while( fgets( buf, 999, svg_fp ) != NULL ) {
 #ifdef WZ
-		if( svg_compress ) gzprintf( outfp, "%s", buf ); 
+		if( svg_compress ) gzprintf( (gzFile) outfp, "%s", buf ); 
 		else 
 #endif
 			printf( "%s", buf );
@@ -749,7 +749,7 @@
 	unlink( svg_tmpfilename );
 #ifdef WZ
 	if( svg_compress && !svg_stdout ) {
-		gzclose( outfp );
+		gzclose( (gzFile) outfp );
 		chmod( svg_filename, 00644 );
 		}
 #endif
